toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -41,6 +44,12 @@ func enableVerityPartition(verity []imagecustomizerapi.Verity, imageChroot *safe
 
 	logger.Log.Infof("Enable verity")
 
+	_, span := otel.GetTracerProvider().Tracer(OtelTracerName).Start(ctx, "enable_verity_partition")
+	span.SetAttributes(
+		attribute.Int("verity_count", len(verity)),
```

Comment:
It might be good to log the name of the verity device, as it will tell us what type of verity is being used.

Reply:
I'd log the list of `imagecustomizerapi.Verity.Name` fields. You could also add list of the `MountPath` fields.

Reply:
updated. I actually figured its better to log this info in `customizeVerityImageHelper` as that is where we provision verity and the time changes based on root/usr. Let me know if you think thats fine.